lib/segment/src/fixtures/sparse_fixtures.rs

```diff
@@ -84,6 +84,9 @@ pub fn fixture_sparse_index_ram<R: Rng + ?Sized>(
         num_vectors
     );
 
+    // assert no points are indexed following open for RAM index
+    assert_eq!(sparse_vector_index.indexed_vector_count(), 0);
```

Comment:
this is the first bug, the assertion failed before the test

Reply:
I like that you covered this changes in tests